registry/default/blocks/post-page/mock-data.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from "react";
+import { ReactNode } from "react";
+
+import { ContentCard } from "@/registry/default/blocks/content-card";
+
+interface MockPost
+  extends Omit<React.ComponentProps<typeof ContentCard>, "children"> {
+  id: string;
+  content: ReactNode;
```

Comment:
this looks like a left over, every mock data represents data coming from the model, there is no react node on the model